<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Communicator</class>
 <widget class="QMainWindow" name="Communicator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>775</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1175</width>
    <height>775</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AeroQuad Communicator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="commuicator.qrc">
    <normaloff>:/icons/resources/AeroQuadIcon.ico</normaloff>:/icons/resources/AeroQuadIcon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="separator"/>
   <addaction name="actionStatus"/>
   <addaction name="actionRoute"/>
   <addaction name="separator"/>
   <addaction name="actionSetup"/>
   <addaction name="actionMotors"/>
   <addaction name="actionCalibrate"/>
   <addaction name="actionFirmware"/>
   <addaction name="separator"/>
   <addaction name="actionPlots"/>
   <addaction name="actionTerminal"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);</string>
   </property>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/AQConnect.svg</normaloff>:/images/resources/AQConnect.svg</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/AQState.svg</normaloff>:/images/resources/AQState.svg</iconset>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/AQMainMenu.svg</normaloff>:/images/resources/AQMainMenu.svg</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/AQCal.svg</normaloff>:/images/resources/AQCal.svg</iconset>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/AQConfig.svg</normaloff>:/images/resources/AQConfig.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/AQTerminal.svg</normaloff>:/images/resources/AQTerminal.svg</iconset>
   </property>
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionPlots">
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/AQPlot.svg</normaloff>:/images/resources/AQPlot.svg</iconset>
   </property>
   <property name="text">
    <string>Plots</string>
   </property>
  </action>
  <action name="actionFirmware">
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/AQupload.svg</normaloff>
     <normalon>:/images/resources/AQupload.svg</normalon>:/images/resources/AQupload.svg</iconset>
   </property>
   <property name="text">
    <string>Firmware</string>
   </property>
   <property name="toolTip">
    <string>Upload Firmware</string>
   </property>
  </action>
  <action name="actionRoute">
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/AQGlobe.svg</normaloff>:/images/resources/AQGlobe.svg</iconset>
   </property>
   <property name="text">
    <string>Route</string>
   </property>
   <property name="toolTip">
    <string>Upload navigation routes</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/debug.png</normaloff>:/images/resources/debug.png</iconset>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <property name="toolTip">
    <string>Test Panel</string>
   </property>
  </action>
  <action name="actionMotors">
   <property name="icon">
    <iconset resource="commuicator.qrc">
     <normaloff>:/images/resources/motor.png</normaloff>:/images/resources/motor.png</iconset>
   </property>
   <property name="text">
    <string>Motors</string>
   </property>
   <property name="toolTip">
    <string>Test Motors</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="commuicator.qrc"/>
 </resources>
 <connections/>
</ui>
